changelog/fragments/1685542803-Tamper-protected-Endpoint-integration-removal.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Tamper protected Endpoint integration removal
```

Comment:
Do we need the changelog entry as well? Seems rather vague, I don't know if that will help a user reading the release notes to know what this really adds.

Reply:
I will combine these 3 change log files into one incorporating all the suggested changes.